cmd/buildah/push.go

```diff
@@ -41,6 +45,8 @@ type pushOptions struct {
 	encryptLayers      []int
 }
 
+var flags *flag.FlagSet
```

Comment:
Why move this variable into the package scope?

Reply:
I need to use it with `if flags.Changed("compression-level") {` later or to check whether the compressionLevel (which is a `*int`) was set.

Reply:
Use `c.Flag("compression-level").Changed` instead?

Reply:
thanks for the hint, that looks better.  I'll fix it